client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -84,56 +69,25 @@ const ScreenMain = ( {
 		}
 	};
 
-	const [ selectedSectionCategory, setSelectedSectionCategory ] = useState( CATEGORY_ALL_SLUG );
+	const handleNavigatorItemSelect = ( type: string, category: string ) => {
+		onMainItemSelect( type );
+		if (
+			category === selectedCategory ||
+			( shouldOpenCategoryList && category === CATEGORY_ALL_SLUG )
```

Comment:
I think we don't need this line because when we click on Section, by default the `category` will be `CATEGORY_ALL_SLUG`:

```
onClick={ () => handleNavigatorItemSelect( 'section', CATEGORY_ALL_SLUG ) }
```

Reply:
I want to close the category list when you click the `Sections` again as it's the current behavior.